tests/indent/bp/select.bp

```diff
@@ -0,0 +1,60 @@
+foo = select(variant("VARIANT"), {
+    "x86": "my_x86",
+    "x86_64": [
+        "x86",
+        "x64",
+    ],
+    "arm": {
+        some: "aarch",
+        value: "aarch64",
+    },
+    default: 0,
+})
+
+foo = select(
+    variant("VARIANT"),
+    {
+        "x86": "my_x86",
+        "x86_64": [
+            "x86",
+            "x64",
+        ],
+        "arm": {
+            some: "aarch",
+            value: "aarch64",
+        },
+        default: 0,
+    }
+)
+
+foo = select((
+    arch(),
+    os(),
+), {
+    (default, default): [],
+})
```

Comment:
Can you drop this test for the time being? This has been a recurring issue for most languages that have this style of code.

Reply:
What's wrong with keeping it as an expected failure?

Reply:
Noise, and CI load.

Reply:
This is _not_ resolved; we don't want to add new expected failures.

Reply:
But I did remove the XFAIL?